app/_components/footer/Footer.tsx

```diff
@@ -1,8 +1,9 @@
-import { Box, useColorMode } from '@chakra-ui/react';
+import { Box } from '@chakra-ui/react';
+import useColorMode from '@/app/_hooks/useColorMode';
 
 const Footer = () => {
-  const { colorMode } = useColorMode();
-  const isDark = colorMode === 'dark';
+  const { isDark } = useColorMode();
```

Comment:
Layout [client](https://github.com/ed-pilots-network/frontend/blob/dec47a760553a163d360a2ba0938ce2afd88ef85/app/layout.client.tsx#L8) also uses this. Can you update that as well. Thanks